pymobiledevice3/cli/apps.py

```diff
@@ -45,10 +45,10 @@ def uninstall(service_provider: LockdownClient, bundle_id):
 
 
 @apps.command('install', cls=Command)
-@click.argument('ipa_or_app_path', type=click.Path(exists=True))
-def install(service_provider: LockdownServiceProvider, ipa_or_app_path: str) -> None:
-    """ install given .ipa/.app """
-    InstallationProxyService(lockdown=service_provider).install_from_local(ipa_or_app_path)
+@click.argument('ipa_or_app_or_ipcc_path', type=click.Path(exists=True))
```

Comment:
It's time to simplify this:
```suggestion
@click.argument('package', type=click.Path(exists=True))
```